### PR TITLE
add cloud-artifacts as codeowners of /schemas/dbt

### PR DESCRIPTION
**Pull request by MichelleArk**
resolves N/A

### Checklist

- [ ] I have read [the contributing guide](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md) and understand what's expected of me
- [ ] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
- [ ] I have run this code in development and it appears to resolve the stated issue
- [ ] This PR includes tests, or tests are not required/relevant for this PR
- [ ] I have [opened an issue to add/update docs](https://github.com/dbt-labs/docs.getdbt.com/issues/new/choose), or docs changes are not required/relevant for this PR
- [ ] I have run `changie new` to [create a changelog entry](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md#adding-a-changelog-entry)

### CODE REVIEW

File: .github/CODEOWNERS
Comment (MichelleArk) [inline .github/CODEOWNERS:79]: https://github.com/orgs/dbt-labs/teams/cloud-artifacts

Comment (MichelleArk) [reply .github/CODEOWNERS:79]: From the [GH docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax): 
> Users and teams must have explicit write access to the repository, even if the team's members already have access.

@leahwicz - are we okay granting write access to @dbt-labs/cloud-artifacts, assuming they don't have it already?

Comment (leahwicz) [reply .github/CODEOWNERS:79]: Yeah that is fine. We have PR reviews enforced so its not like an accidental commit can be pushed

**Review by leahwicz — APPROVED** [review]

**Comment by MichelleArk** [comment]
@aranke - I don't believe this will block merging on code review from Cloud Artifacts based on the behaviour I've been seeing for changes to files owned by existing codeowners defined. 

Here's a recent example that modified [`core/dbt/adapters/base/impl.py`](https://github.com/dbt-labs/dbt-core/blob/main/.github/CODEOWNERS#L56), tagged `Core - Adapters` as codeowners, but did not require an approval to merge: https://github.com/dbt-labs/dbt-core/pull/7307

I'll create a test PR to confirm the behaviour once this is merged though!

**Pull request merged**
